test/parallel/test-zlib.js

```diff
@@ -69,7 +69,7 @@ if (!process.env.PUMMEL) {
   strategy = [0];
 }
 
-let testFiles = ['person.jpg', 'elipses.txt', 'empty.txt'];
+let testFiles = ['person.jpg', 'ellipses.txt', 'empty.txt'];
```

Comment:
the file should be renamed too

Reply:
Can you please run `git mv test/fixtures/elipses.txt test/fixtures/ellipses.txt` to fix the test failure?